src/vs/workbench/browser/parts/editor/breadcrumbsControl.ts

```diff
@@ -215,6 +219,7 @@ export class BreadcrumbsControl {
 		this._widget.onDidSelectItem(this._onSelectEvent, this, this._disposables);
 		this._widget.onDidFocusItem(this._onFocusEvent, this, this._disposables);
 		this._widget.onDidChangeFocus(this._updateCkBreadcrumbsActive, this, this._disposables);
+		this._widget.onDidRightClick(this._onRightClickEvent, this, this._disposables);
```

Comment:
I think @jrieken would agree that it is better to have context menu rather than implicit right click behavior:

<img src="https://github.com/microsoft/vscode/assets/46503702/6022f3d2-a379-4cfc-b3a8-7042acc14a50" width="480">

Looking at statusbarPart contextmenu implementation it seems to be possible with this:

```suggestion
		this._disposables.add(
			dom.addDisposableListener(this.domNode, dom.EventType.CONTEXT_MENU, e => this._onRightClickEvent(e))
		);
		this._disposables.add(Gesture.addTarget(this.domNode));
		this._disposables.add(dom.addDisposableListener(this.domNode, TouchEventType.Contextmenu, e => this._onRightClickEvent(e)));
```

It worked by also changing _onRightClickEvent to:

```ts
private _onRightClickEvent(e: MouseEvent | GestureEvent) {
    dom.EventHelper.stop(e, true);

    const event = new StandardMouseEvent(e);

    this.contextMenuService.showContextMenu({
        getAnchor: () => ({ x: event.posx, y: event.posy }),
        getActions: () => {
            return [
                toAction({
                    id: 'breadcrumbs.copy', label: localize('cmd.copy', "Copy Breadcrumbs"), run: () => {
                        const textList = this._widget.getTextListAll();
                        const path = textList.join(this._cfSeparator.getValue());
                        this.clipboardService.writeText(path);
                    }
                }),
                toAction({
                    id: 'breadcrumbs.copy', label: localize('breadcrumbs.copyToFocus', "Copy Breadcrumbs to Focus"), run: () => {
                        const textList = this._widget.getTextListUptoNode(event.target);
                        const path = textList.join(this._cfSeparator.getValue());
                        this.clipboardService.writeText(path);
                    }
                })
            ];
        },
        onHide: () => { }
    })

}
```